pkg/features/kube_features.go

```diff
@@ -610,6 +610,15 @@ const (
 	// Enables controlling pod ranking on replicaset scale-down.
 	PodDeletionCost featuregate.Feature = "PodDeletionCost"
 
+	// owner: @ddebroy
+	// alpha: v1.25
+	// beta: v1.26
```

Comment:
beta and ga should be added here when we reach those stages in next releases.

Reply:
indeed - removed beta and ga